src/InputField/index.js

```diff
@@ -124,6 +125,10 @@ const Suffix = styled(({ children, className }) => <div className={className}>{c
   & svg {
     color: ${({ theme }) => theme.orbit.colorIconSecondary};
   }
+  ${StyledServiceLogo} {
```

Comment:
Remove `cursor: pointer` from `Suffix` and add `width: 100%` to `Input`

Always check the storybook, how to component looks and behaves 🙏